validphys2/src/validphys/results.py

```diff
@@ -307,6 +306,27 @@ def procs_data_values_experiment(proc_result_table_experiment):
     "dataset_inputs_results_central", ("group_dataset_inputs_by_experiment",)
 )
 
+groups_results_central = collect(
+    "dataset_inputs_results_central", ("group_dataset_inputs_by_metadata",)
+)
+
+
+def group_result_central_table_no_table(groups_results_central, groups_index):
+    """Generate a table containing the data central value and the central prediction"""
+    result_records = []
+    for group_results in groups_results_central:
+        dt, th = group_results
+        for dt_central, th_central in zip(dt.central_value, th.central_value):
+            result_records.append(
+                dict([("data_central", dt_central), ("theory_central", th_central)])
+            )
```

Comment:
```suggestion
            result_records.append({"data_central": dt_central, "theory_central": th_central})
```

I think you can get away with this since you don't need the extra item per replica of the total table (but ofc check please)

An easy test to add would be to check that indeed calling `group_result_central_table_no_table` returns the central value of `group_result_table_no_table`